.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches: [ staging, trying, master ]
+  pull_request:
+
+name: CI
+
+jobs:
+  ci-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # All generated code should be running on stable now
+        rust: [stable]
+
+        include:
+          # Test MSRV
+          - rust: 1.38.0
+
+          # Test nightly but don't fail
+          - rust: nightly
+            experimental: true
```

Comment:
By itself this won't do anything, `experimental` isn't magic (I think `wg/ops/post-transfer.md` is wrong here). You need to set `continue-on-error` to [match against your flag](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error), and (probably?) set `experimental` to `false` for the other jobs.

Reply:
Yeah, that's odd, it was missing from the template. I opened https://github.com/rust-embedded/wg/pull/518 to fix that.